p2p/server.go

```diff
@@ -175,7 +175,7 @@ func (serv *ExchangeServer[H]) handleRequestByHash(hash []byte) ([]H, error) {
 
 	span.AddEvent("fetched-header-from-store", trace.WithAttributes(
 		attribute.String("hash", header.Hash(hash).String()),
-		attribute.Int64("height", h.Height())),
+		attribute.Int64("height", int64(h.Height()))),
```

Comment:
eventually would be good to break metrics too to have it be consistent.

Reply:
I wanted to, but there was no uint64 attrb